src/modm/platform/uart/at90_tiny_mega/module.lb

```diff
@@ -84,20 +84,28 @@ def prepare(module, options):
         ":architecture:uart",
         ":platform:gpio")
 
-    driver = device.get_driver("usart")
+    driver = device.get_driver("usart:avr")
     if "instance" in driver:
         for instance in listify(driver["instance"]):
             module.add_submodule(Instance(int(instance)))
     else:
-        return False
+        load_options(module)
```

Comment:
Disclaimer: I don't know what this does, I just copied it from the SPI implementation you pointed at.

Reply:
That calls the [function defined in L22](https://github.com/modm-io/modm/blob/develop/src/modm/platform/uart/at90_tiny_mega/module.lb#L22) that just adds the Buffer options. It's just a separate function so you can use it for both the `modm:platform:uart` as well as its submodules.